tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -62,3 +62,4 @@ spec:
       kubectl set env daemonset/aws-node -n kube-system ENABLE_PREFIX_DELEGATION=true
       # install csi drivers.
       kubectl apply -k "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=$(params.aws-ebs-csi-driver-version)"
+      kubectl scale --replicas 20 deploy coredns -n kube-system
```

Comment:
Can you add a todo to revisit this ?
This solution doesn't scale as we increase our cluster size. We might need to think how we can calculate this number dynamically instead of a static count. We have a doc on this to tackle it multiple ways.

Reply:
Can you share the doc link here ?

Reply:
Have added a TODO.

Reply:
Can you link the doc here ?